composer.json

```diff
@@ -22,6 +22,7 @@
     "require": {
         "php": "^7.1",
         "doctrine/orm": "^2.6",
+        "doctrine/doctrine-bundle": "^1.12 || ^2.0",
```

Comment:
Can you explain why we need this new dependency? Also, it looks like the dependency to `doctrine/persistence` is not explicit, which may cause trouble in the future.

Reply:
We need this here: https://github.com/sonata-project/SonataDashboardBundle/blob/63f484fd8ca96693117e46610c2c2d9f1510355f/src/Resources/config/orm.xml#L11